gooddata-java/src/main/java/com/gooddata/sdk/service/account/AccountService.java

```diff
@@ -24,6 +25,10 @@
  */
 public class AccountService extends AbstractService {
 
+    public static final UriTemplate TEMPLATE = new UriTemplate(Account.URI);
```

Comment:
Just rename to `ACCOUNT_TEMPLATE`

Reply:
This is a good question - I left the name `TEMPLATE` on all the places where the prefix would be the same as the prefix of the `Service`. In other words I added prefixes only in case there were more templates in one service... 
@peter-plochan do you have an opinion on this?